dart/examples/io_example/main.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:sentry/sentry.dart';
+
+import 'event_example.dart';
+
+/// Sends a test exception report to Sentry.io using this Dart client.
+Future<void> main(List<String> rawArgs) async {
+  if (rawArgs.length != 1) {
```

Comment:
I'd just hardcode the DSN here as the other samples, so we can run directly from IDE without code changes or params

Reply:
✅